src/stories/MyButton.vue

```diff
@@ -18,7 +18,7 @@ const props = withDefaults(defineProps<{
   /**
```

Comment:
Make namings consistent `button.css, Button.stories.ts, MyButton.vue, SearchByComponent`. 
1. If you have one component with name `Component` than other also should have it. 
2. If you have MyButton.vue than css for it and stories should also have MyButton